install/upgrade.go

```diff
@@ -156,3 +159,47 @@ func upgradeDeployment(ctx context.Context, k *K8sInstaller, params upgradeDeplo
 	*patched++
 	return nil
 }
+
+func (k *K8sInstaller) UpgradeWithHelm(ctx context.Context, k8sClient genericclioptions.RESTClientGetter) error {
+	if k.params.ListVersions {
+		return k.listVersions()
+	}
+	if err := k.preinstall(ctx); err != nil {
+		return err
+	}
+
+	// TODO (ajs): gethelmValues "extends" many Helm-parameters based on input flags in a non-obvious way, consider
+	//  removing it in favor of documentation. For now, it's needed for consistency with the install commands.
```

Comment:
not sure if this is necessary. you can use `--reuse-values` flag right?

Reply:
Not sure if `gethelmValues` is necessary, or removing it is necessary? One of the things it does is set the container image version for all components based on the `--version` flag.